src/i18n/TranslationLoader.js

```diff
@@ -0,0 +1,16 @@
+import * as messages from './messages';
+
+export const DEFAULT_LOCALE = 'en';
+
+export const resolveBrowserLocale = () => {
+    // from http://blog.ksol.fr/user-locale-detection-browser-javascript/
+    // Rely on the window.navigator object to determine user locale
+    const { language, browserLanguage, userLanguage } = window.navigator;
+    return (language || browserLanguage || userLanguage || DEFAULT_LOCALE).split('-')[0];
+};
+
+export const loadTranslations = locale => (
```

Comment:
This implementation isn't future-proof. When we've got 20 locales, we'll load all 20 dictionaries in the browser, and only choose one. That'll make the bundle heavier for no good reason.

Let's keep it like that for now, but it'll have to be updated.